data/libs/functions/player.lua

```diff
@@ -554,26 +554,34 @@ function Player.updateHazard(self)
 	return true
 end
 
-function Player:addItemStoreInbox(itemId, amount, moveable)
+function Player:addItemStoreInboxEx(item, moveable, setOwner)
```

Comment:
isn't better to use 'movable' than 'moveable'?

Reply:
they're both correct, we use moveable more often than movable so I'm being consistent here

Reply:
I'll open a separate pr after this to change the spelling on everything